src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -22,8 +22,16 @@
 
   <ItemGroup>
     <PackageReference Include="LibLog" Version="5.0.6" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net461" Version="1.0.0">
```

Comment:
We already include the .NET Framework reference assemblies [here](/DataDog/dd-trace-dotnet/blob/7fcc5eec1b3e716b90bf0c5f2f1ebae4fa150d76/src/Directory.Build.props#L32).

Reply:
Accidental commit.